server/container_create_linux.go

```diff
@@ -491,6 +491,13 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID, contai
 			specgen.SetProcessOOMScoreAdj(int(resources.GetOomScoreAdj()))
 			specgen.SetLinuxResourcesCPUCpus(resources.GetCpusetCpus())
 			specgen.SetLinuxResourcesCPUMems(resources.GetCpusetMems())
+
+			hugepageLimits := resources.GetHugepageLimits()
+			if len(hugepageLimits) != 0 {
```

Comment:
is the `if len(hugepageLimits) != 0` check redundant?

Reply:
Yes remove this the range command below will handle this situation.

Reply:
done